src/garage/torch/q_functions/continuous_mlp_q_function.py

```diff
@@ -0,0 +1,40 @@
+"""This modules creates a continuous MLP Q fcuntion network."""
+
+
+class ContinuousMLPQFunction:
+    """
+    This class implements a q value network.
+
+    It predicts the q value for all actions based on the input state. It uses
+    an MLP to fit the function of Q(s, a).
+    """
+
+    def __init__(self,
+                 env_spec,
+                 nn_module,
+                 name='ContinuousMLPQFunction',
+                 input_include_goal=False):
+        """
+        Initialize class with multiple attributes.
+
+        Args:
+            env_spec():
+            nn_module():
+                A PyTorch module.
```

Comment:
I don't think these need to be newline/indented? don't do that unless you go over the column limit